communication/nats_discovery/address_test.go

```diff
@@ -40,6 +40,7 @@ func TestNewAddressForContact(t *testing.T) {
 		Type: "nats/v1",
 		Definition: ContactNATSV1{
 			Topic: "123456",
+			BrokerAddresses: []string{"nats://" + natsServerIp + ":4222"},
```

Comment:
Dont depend on live variables.
Just pass mocked variable like `"nats://far-server:1234"` and assert that it was parsed correctly